docs/stencil-docs/storefront-customization/custom-templates.md

```diff
@@ -149,6 +151,9 @@ title: Custom template - custom-brand.html
 
 <!-- type: tab-end -->
 
+<!-- theme: warning -->
+> #### Stencil CLI versus storefront
+> When adding front matter directly to a custom template, Stencil CLI behaves as explained above. You must add front matter to the default template. However, when applying a theme to the storefront, it is possible to specify front matter directly to the custom template.
```

Comment:
Stencil CLI starting from version `5.2.1` has the same behaviour as a regular mode (eg. live store) - both refular and CLI modes use front matter from custom template, not default one.